app/Services/Eggs/Sharing/EggExporterService.php

```diff
@@ -42,36 +54,40 @@ public function handle(int $egg): string
         $egg = $this->repository->getWithExportAttributes($egg);
 
         $struct = [
-            '_comment' => 'DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO',
             'meta' => [
-                'version' => 'PTDL_v1',
+                'version' => 'PTDL_v2',
```

Comment:
Apart from switching from JSON to YAML, this didn't really change the format, did it? Considering JSON is valid YAML, do we even have to change the version here?

The only change I can think of right now are the fields that used to json strings.

Reply:
The json fields are the only reason I changed it. Because you can still upload old json eggs I wanted to have support for migrating the fields to yaml and decided on a version change, but we could also just transform the fields to yaml when a json egg is uploaded.

Reply:
I think v2 is fine here, it isn't used for much and might be useful down the road.